dataactbroker/handlers/fileHandler.py

```diff
@@ -644,7 +650,12 @@ def generateD2File(self):
         # Generate and upload d2 file to S3
         user_id = LoginSession.getName(session)
         timestamped_name = s3UrlHandler.getTimestampedFilename(CONFIG_BROKER["award_file_name"])
-        upload_file_name = "".join([str(user_id), "/", timestamped_name])
+
+        if self.isLocal:
+            upload_file_name = "".join([CONFIG_BROKER['broker_files'], timestamped_name])
```

Comment:
Should this incorporate the `user_id` at all? Ideally we'd have a very similar set up between local and AWS.